play/speedy-math2/js/app.js

```diff
@@ -209,12 +229,17 @@ function newProblem() {
     let [ operandOne, operandTwo ] = getOperands(operation);
 
     // set the problem display to the newly generated problem
-    problemEl.innerHTML = `${operandOne} ${operation.sign} ${operandTwo} = ?`;
+    if (operation.type != "exponent") {
+      problemEl.innerHTML = `${operandOne} ${operation.sign} ${operandTwo} = ?`;
+    }
+    else {
+      // TODO Display exponent problem with special formatting
```

Comment:
You'll most likely have to use the ```<sup>``` tag, which is nice for formatting, but it'll be annoying to strip out when you are trying to calculate the answer.

Reply:
Thanks for the heads up